pkg/client/load-test.go

```diff
@@ -48,14 +48,20 @@ func (apiLoadTester ArmadaLoadTester) RunSubmissionTest(ctx context.Context, spe
 			wg.Add(1)
 			go func(i int, submission *domain.SubmissionDescription) {
 				defer wg.Done()
-				jobIdChannel, jobSetId := apiLoadTester.runSubmission(ctx, submission, i)
+
+				nestedWg := &sync.WaitGroup{}
```

Comment:
I would name this according to what it is for i.e `submissionWg`, `jobSubmissionWg`, or something like that.